chapters/memory-security/ctf/reading/README.md

```diff
@@ -0,0 +1,13 @@
+# CTF
+
+In this laboratory, you will have to apply most of the concepts presented throughout this course under the format of `Capture-The-Flag` tasks.
+These tasks will test your understanding and mastery of specific static and dynamic analysis methods and tools, the compilation process, assembly language - syntax, registers, memory handling, functions, - as well as your ability to identify and exploit simple buffer overflow vulnerabilities.
+
+## Return Oriented Programming
```

Comment:
You need to expand on what gadgets are and what ROPs do. I created the image below for a ROP demo I made the other day. You can use it as a start. Explain what happens step by step, which value on the stack goes where and it should be clearer.

![rop-anatomy](https://github.com/user-attachments/assets/a9540377-0ce5-4b36-98f3-9add6ce1f8da)

Reply:
Bump on this though. This would be of great use. You already have the image above, just add some text to walk students through it.